1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,88 @@
 // Write your code here
+
+let index = 0;
+function basicStructure(array) {
+  //Creating containers
+  const parentDiv = document.createElement("div");
```

Comment:
Code works (apart from the auro scrolling stuff), and comments are a nice touch but  I do have some complants I'm afraid. Honestly, the choice of var names makes this code hard to follow, and the function is quite large (I would break it out into smaller functions with descriptive names that do one thing each), that's OK, most code starts out this way and it works which is I guess what really matter. However, it needs a little refactoring to be nice and tidy production ready code, which is worth doing for practice if you have the time